shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/main/java/org/apache/shardingsphere/shadow/rule/ShadowRule.java

```diff
@@ -51,43 +56,95 @@
     
     private final boolean enable;
     
-    private final Map<String, ShadowDataSourceConfiguration> dataSources = new LinkedHashMap<>();
-    
-    private final Map<String, ShadowTableConfiguration> tables = new LinkedHashMap<>();
+    private final Map<String, String> shadowDataSourceMappings = new LinkedHashMap<>();
     
     private final Map<String, ShadowAlgorithm> shadowAlgorithms = new LinkedHashMap<>();
     
+    private final Map<String, ShadowTableRule> shadowTableRules = new LinkedHashMap<>();
+    
     public ShadowRule(final ShadowRuleConfiguration shadowRuleConfig) {
+        enable = shadowRuleConfig.isEnable();
```

Comment:
upon the boolean variable style is `isXXXX`, but here enable is not `isEnable` . why not use same varible style ?

Reply:
isXXXX is added by lombok annotation @getter.